File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length() != t.length()) return false;
        int[] freq1 = new int[26];
        for(char ch:s.toCharArray()){
            freq1[ch-'a']++;
        }
        int[] freq2 = new int[26];
        for(char ch:t.toCharArray()){
            freq2[ch-'a']++;
        }
        
        return Arrays.equals(freq1,freq2);
    }
}

File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        int[] freqS = new int[26];
        int[] freqT = new int[26];
        foreach(var ch in s){
            freqS[ch-'a']++;
        }
        foreach(var ch in t){
            freqT[ch-'a']++;
        }
        return freqS.SequenceEqual(freqT);
    }
}